webrecorder/browsertrix
# Display & edit crawl schedule in user local time

(https://github.com/webrecorder/browsertrix-cloud/issues/255) Addresses crawl scheduling timezone issues:
- Show schedule in crawl detail view in local time
- Set initial schedule edit time to time already specified in schedule
- Show schedule in crawl template list

### Manual testing
1. Run `yarn` to update dependencies and then run app
2. Go to crawl template list. Verify schedule and tooltip with next crawl date displays correctly
3. Click "..." actions menu -> "Edit crawl schedule" for a crawl template that already has a schedule. Verify time is displayed correctly.
4. Edit schedule. Verify schedule saves and updates as expected.
5. Go to crawl template detail. Verify "Recurring crawls" field displays schedule as expected.
6. Repeat edit test from detail page

### Screenshots
**Edit schedule form:**
<img width="527" alt="Screen Shot 2022-06-21 at 8 18 20 PM" src="https://user-images.githubusercontent.com/4672952/174925800-33784942-7b90-4245-aaec-e86f1784dc26.png">

**Local schedule in crawl template detail view:**
<img width="253" alt="Screen Shot 2022-06-21 at 8 18 34 PM" src="https://user-images.githubusercontent.com/4672952/174925804-67cb75bf-b3ea-4e84-b9bd-7cf592273677.png">
<img width="353" alt="Screen Shot 2022-06-21 at 8 18 43 PM" src="https://user-images.githubusercontent.com/4672952/174925805-2d75fc04-977b-4920-a74e-196cf69bfc1c.png">
<img width="281" alt="Screen Shot 2022-06-21 at 8 19 07 PM" src="https://user-images.githubusercontent.com/4672952/174925806-23b59772-f63e-4514-8cbe-c3838d163db6.png">

**Schedule in crawl template list view with tooltip:**
<img width="1023" alt="Screen Shot 2022-06-21 at 8 47 18 PM" src="https://user-images.githubusercontent.com/4672952/174925807-14e55101-b497-4694-90e4-63b4c3e2f5d9.png">
